pkg/provider/data_source_cluster.go

```diff
@@ -288,9 +290,32 @@ func (c *clusterDataSource) Schema(ctx context.Context, req datasource.SchemaReq
 				MarkdownDescription: "Earliest backup recover time.",
 				Computed:            true,
 			},
+			"maintenance_window": schema.SingleNestedAttribute{
+				Computed:            true,
+				MarkdownDescription: "Custom maintenance window.",
+				Attributes: map[string]schema.Attribute{
+					"is_enabled": schema.BoolAttribute{
+						MarkdownDescription: "Is maintenance window enabled.",
+						Computed:            true,
+					},
+					"start_day": schema.Float64Attribute{
+						MarkdownDescription: "Start day.",
+						Computed:            true,
+						Validators: []validator.Float64{
+							startDayValidator(),
+						},
```

Comment:
This validator checks only if the value is between 0 and 6. For readability and simplicity, we can directly write it. I would propose something like:
```suggestion
						Validators: []validator.Float64{
							float64validator.Between(0, 6),
						},
```
Directly moving this function here. WDYT, @wai-wong-edb ?

Reply:
sure that is simpler